content/contribute/index.md

```diff
@@ -7,9 +7,10 @@ save_as: contribute/index.html
 ---
 
 So you want to contribute to IPFS and the ecosystem? Here is a quick listing
-of everything we need help with and how you can get started helping out. Please
-note that even though what you want to do is not listed here, we probably accept
-contributions for it! If you're unsure, please open a issue.
+of things we need help with and how you can get started. Even if what you want
+to do is not listed here, we probably accept contributions for it! If you're
+unsure, please open a issue.
```

Comment:
Should include direction to where to open up a issue. I'm leaning towards pointing people to open issue on ipfs/community.